.env.example

```diff
@@ -1,9 +0,0 @@
-NODE_ENV="development"
```

Comment:
Why was this .env.example file deleted?

Reply:
I mistakenly renamed .env.example to .env(which is added to .gitignore) to make creating of schema and running migrations available on my machine. In this case new file called `.env` should be created on base of `.env.example` instead of renaming.

Reply:
You should make a commit that brings this file back, so that after merging this file is available in the dev branch.